test/mocha.opts

```diff
@@ -1,4 +1,5 @@
 --require babel-register
+--require jsdom-global/register
```

Comment:
What is this for, don't we already inject what's needed in https://github.com/callemall/material-ui/blob/v1-alpha/test/utils/dom.js?